contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -33,6 +33,8 @@ import {
 
 import { RewardsMath } from "./RewardsMath.sol";
 
+uint32 constant SECONDS_PER_DAY = 60 * 60 * 24;
```

Comment:
I believe you can just use `1 day` here or just get rid of it and do the multiplication later by `1 day`

Reply:
Yes I can

Reply:
done